### ISSUE TITLE
How can I add RESOLUTION in m3u8 playlist?

### ISSUE THREAD

**Issue by aungmyokyaw97** [pascalbaljetmedia/laravel-ffmpeg#175]
I am using  this package.

$lowBitrate = (new \FFMpeg\Format\Video\X264('aac','libx264'))->setKiloBitrate(250);
$midBitrate = (new \FFMpeg\Format\Video\X264('aac','libx264'))->setKiloBitrate(500);

FFMpeg::fromDisk('public')
->open('test.mp4')
->exportForHLS()
->setSegmentLength(8)
->toDisk('public')
->addFormat($lowBitrate)
->addFormat($midBitrate)
->save('test/test.m3u8');

When I export,I got like this 

`#EXTM3U`
`#EXT-X-STREAM-INF:BANDWIDTH=250000`
`test_250.m3u8`
`#EXT-X-STREAM-INF:BANDWIDTH=500000`
`test_500.m3u8`

Here,I want like this
`#EXTM3U`
`#EXT-X-STREAM-INF:BANDWIDTH=250000,RESOLUTION=398x224`
`test_250.m3u8`
`#EXT-X-STREAM-INF:BANDWIDTH=500000,RESOLUTION=480x270`
`test_500.m3u8`

Any suggestion are welcome!

**Comment by pascalbaljet** [comment]
I've rewritten the package and it has support for this. Could you test the `v7` branch? Almost no breaking changes except for some filter stuff in HLS exports.

https://github.com/pascalbaljetmedia/laravel-ffmpeg/tree/v7

`composer require "pbmedia/laravel-ffmpeg:v7.x-dev"`

**Comment by shanka12** [comment]
This is works with v7. I got like this

```
#EXTM3U
#EXT-X-STREAM-INF:BANDWIDTH=243637,RESOLUTION=480x360,FRAME-RATE=25.000
3845550697_0_250.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=465427,RESOLUTION=640x480,FRAME-RATE=25.000
3845550697_1_500.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=927805,RESOLUTION=1280x720,FRAME-RATE=25.000
3845550697_2_1000.m3u8
#EXT-X-ENDLIST
```

**Comment by pascalbaljet** [comment]
Fixed in `v7.0`
